src/offchainapi/errors.py

```diff
@@ -14,26 +14,30 @@ class OffChainErrorCode(Enum):
 
 
     # Indicates that a request with the same cid has already been received,
```

Comment:
received -> committed
```suggestion
    # Indicates that a request with the same cid has already been committed,
```